api/namex/resources/name_requests/name_request.py

```diff
@@ -54,10 +54,11 @@ def get(self, nr_id):
             if nr_model.stateCd == 'DRAFT':
                 oldest_draft = Request.get_oldest_draft().submittedDate
```

Comment:
i think this would throw an error if Request.get_oldest_draft() returned None.

Reply:
Fixed as per comment